### ISSUE TITLE
Missing Alt Text for book covers

### ISSUE THREAD

**Issue by binkylush** [psu-libraries/psulib_blacklight#1379]
On both the search results page and the single item display page, the book covers do not have alt text. Attached are screenshots of the wave results for these pages:

[https://catalog.libraries.psu.edu/catalog/2446312](https://catalog.libraries.psu.edu/catalog/2446312)
[https://catalog.libraries.psu.edu/?f%5Baccess_facet%5D%5B%5D=In+the+Library&search_field=all_fields&q=dahl](https://catalog.libraries.psu.edu/?f%5Baccess_facet%5D%5B%5D=In+the+Library&search_field=all_fields&q=dahl)

![Image](https://github.com/user-attachments/assets/0b6d8242-1c96-4a4d-9e8d-4da7c89e097b)

![Image](https://github.com/user-attachments/assets/b965f243-dc63-4b1c-8b7d-f0bbd7e31736)

**Comment by Smullz622** [comment]
alt text should be '(book title) book cover'

**Comment by memsdm05** [comment]
I am following [Penn State's accessibility guide](https://accessibility.psu.edu/images/imageshtml/)

**Comment by ruthtillman** [comment]
Ran across one issue where the book has quotes in the title: https://catalog.qa.k8s.libraries.psu.edu/catalog/43928624

> "Are you calling me a racist?" : why we need to stop talking about race and start making real antiracist change / Sarita Srivastava

wreaks havoc on the alt text:

> <img class="img-fluid" src="https://books.google.com/books/content?id=EpvqEAAAQBAJ&amp;printsec=frontcover&amp;img=1&amp;zoom=5&amp;edge=cur1" alt="Cover image for " are="" you="" calling="" me="" a="" racist?""="">

Looks like we need one minor change where the quotes are handled by: removal, replacement with a single quote, escaping
